test/e2e-oidc/external_oidc.go

```
@@ -0,0 +1,1239 @@
+	orig := cm2.DeepCopy()
+	cm2.Data["auth-config.json"] = "manually overwritten"
+	cm2, err = testClient.kubeClient.CoreV1().ConfigMaps(managedNS).Update(testCtx, cm2, metav1.UpdateOptions{})
+	require.NoError(t, err)
+	require.NotEqual(t, cm2.Data, orig.Data)
+
+	// wait for CAO to overwrite it
+	var cmErr2 error
+	waitErr2 := wait.PollUntilContextTimeout(testCtx, 2*time.Second, 1*time.Minute, false, func(ctx context.Context) (bool, error) {
+		cm2, cmErr2 = testClient.kubeClient.CoreV1().ConfigMaps(managedNS).Get(testCtx, authCM, metav1.GetOptions{})
+		if cmErr2 != nil {
+			return false, nil
```

golangci-lint (2.12.2)

[error] error is not nil (line 355) but it returns nil

(nilerr)